.github/workflows/ci-workflow.yml

```diff
@@ -23,6 +23,20 @@ jobs:
       - name: test
         run: CC=gcc CXX=g++ make test
 
+  gcc-make-debug:
+    name: gcc make debug
+    needs: [ clang-formatting-check ]
```

Comment:
We should make this depend on `gcc-build-test` instead. If the gcc build test does not success we should not run the debug build.

Reply:
yes added that